libraries/mysql_service_manager_sysvinit.rb

```diff
@@ -39,7 +39,7 @@ class MysqlServiceManagerSysvinit < MysqlServiceBase
       end
 
       service mysql_name do
-        provider Chef::Provider::Service::Init::Redhat if node['platform_family'] == 'redhat'
+        provider Chef::Provider::Service::Init::Redhat if %w(redhat suse).include?(node['platform_family'])
```

Comment:
Also I fixed the platform family bit. That was entirely broken for a long time clearly.